lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -17,6 +17,9 @@ package restapi
 import (
 	"context"
 	"crypto/tls"
+	"encoding/json"
+	errs "errors"
```

Comment:
Inconsistent with how we have imported "errors" without the alias in all other files.